### ISSUE TITLE
Test output is black

### ISSUE THREAD

**Issue by ZH3nGsQ** [IceClear/StableSR#133]
I want to test StableSR using your test_example, the result is a black image.
I download pretrained_SDmodel `v2-1_512-ema-pruned.ckpt` ,pretrained_CFWmodel `vqgan_cfw_00011.ckpt` and `stablesr_000117.ckpt`.
And I run the code `python scripts/sr_val_ddpm_text_T_vqganfin_oldcanvas_tile.py --config configs/stableSRNew/v2-finetune_text_T_512.yaml --ckpt pretrained_turbo/stablesr_000117.ckpt --init-img inputs/test_example --outdir OUT_PATH --ddpm_steps 4 --dec_w 0.5 --seed 42 --n_samples 1 --vqgan_ckpt pretrained_CFWmodel/vqgan_cfw_00011.ckpt --colorfix_type wavelet --upscale 1`
The input is 720*720
![image](https://github.com/IceClear/StableSR/assets/163538949/c249f798-68b1-4945-8d26-862c78819652)
The output is also 720*720 but black
![image](https://github.com/IceClear/StableSR/assets/163538949/2abca1ef-7118-4039-b1c7-c5b03bd49842)
I don't know which part went wrong.

**Comment by ZH3nGsQ** [comment]
I changed ckpt_path to pretrained_SDmodel/v2-1_512-ema-pruned.ckpt in v2-finetune_text_T_512.yaml

**Comment by IceClear** [comment]
Hi. I did not meet such a problem. You may check if the output is None. If so, you may need to avoid using autocasting for the autoencoder part. This is not very common.
